bots/discord/game-server-watcher/README.md

```diff
@@ -0,0 +1,18 @@
+# Game Server Watcher
+
+## From [Github](https://github.com/a-sync/game-server-watcher#readme)
+A simple discord/telegram/slack bot that can be hosted on a free service to monitor your game servers and players in style. 😎  
+
+## Screenshots
+<img src="https://user-images.githubusercontent.com/14183614/162092529-e1645b44-2650-4893-8123-7ba187b1f51c.png" height="280"> <img src="https://user-images.githubusercontent.com/14183614/162092488-f28bd60c-88bf-4b1e-a31e-d7dca51d8c28.png" height="280"> <img src="https://github.com/a-sync/game-server-watcher/assets/14183614/0461ad76-bb13-468c-a7b3-437d6a3cea63" height="280"> <img src="https://github.com/a-sync/game-server-watcher/assets/14183614/ee0ef0de-83bc-42ae-8f64-62f481f6ba8f" height="280"> 
+
+## Running the bot
+Refer to the wiki on how to acquire tokens for:
+ * [steam](https://github.com/a-sync/game-server-watcher/wiki/Steam-Web-API-key)
+ * [discord](https://github.com/a-sync/game-server-watcher/wiki/Discord-bot-token)
+ * [telegram](https://github.com/a-sync/game-server-watcher/wiki/Telegram-bot-token)
+ * [slack](https://github.com/a-sync/game-server-watcher/wiki/Slack-bot-token)
+
+## Server Ports
```

Comment:
No no the ports it uses must be in that form, variables not: see https://github.com/parkervcp/eggs/tree/master/example

Reply:
ahh. i see what you mean :)
so something like:  
---

## Server Ports
GSW Control Panel requires one port for http traffic.  
(It can be any port.)  

| Port    | default |
|---------|---------|
| Web     | 8080    |

---
note: in the startup script im using `env PORT={{SERVER_PORT}}` to set the env var directly so it should always default to whatever port is assigned to the server.